lib/private/User/Manager.php

```diff
@@ -598,4 +602,14 @@ public function getByEmail($email) {
 			return ($u instanceof IUser);
 		}));
 	}
+
+	private function verifyUid(string $uid) {
```

Comment:
```suggestion
	private function verifyUid(string $uid): bool {
```

but I guess we should backport this until 14? so we should remove `string ` actually instead?